<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DES工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Encrypt_btn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>加密</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Decrypt_btn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>解密</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>明文</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>加密密钥</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>密文</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Key_text">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Plain_text">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>361</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Key_gen_btn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>生成密钥</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Cipher_text">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>361</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Plain_text2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>361</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Key_text2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>解密密钥</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>明文</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Key_copy_btn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>复制密钥</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Addition_text">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>300</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>补充位数</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
